pyproject.toml

```diff
@@ -46,7 +46,7 @@ djangocms-apphook-setup = "0.4.1"
 djangocms-attributes-field = "2.1.0"
 djangocms-blog = "^1.2"
 # djangocms-bootstrap4 commit to remove deprecated imports: https://github.com/django-cms/djangocms-bootstrap4/pull/162
-djangocms-bootstrap4 = { git = "https://github.com/django-cms/djangocms-bootstrap4", rev = "2fc7f14c2b7b53882faf6830289b5052f775965f" }
+djangocms-bootstrap4 = { git = "https://github.com/django-cms/djangocms-bootstrap4", rev = "b17bad6ad755cd40a3062e6ac692419136c27630" }
```

Comment:
This is installing the `main` branch merge of https://github.com/django-cms/djangocms-bootstrap4/pull/162, instead of a commit within it.

Reply:
No… don't do this now. Maybe it breaks something. This PR shouldn't be to blame.